examples/tts/g2p/conf/heteronym_classification_zh.yaml

```diff
@@ -0,0 +1,104 @@
+name: HeteronymClassification
+
+# Dataset info
+train_manifest: ???
+validation_manifest: ???
+test_manifest: ???
+do_training: True
+do_testing: False
+pretrained_model: null # path to .nemo file or model name from list_available_models()
+
+model:
+  wordids: ??? # path to CPP wordids in WikiHomograph dataset format
```

Comment:
could add link to the dataset here please? and `Chinese Polyphones with
Pinyin (CPP)`